frontend/src/components/AppHeader/AppHeader.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '../../test-utils';
+import { screen } from '@testing-library/react';
+import { AppHeader } from './AppHeader';
+
+describe('AppHeader', () => {
+  it('displays page title', () => {
+    render(<AppHeader pageTitle="Home" />, {});
+    expect(screen.getByText('Home')).toBeInTheDocument();
```

Comment:
Tutaj ciekawostka, choć to też dobrze.
Gdy użyjesz samego:
`screen.getByText('Home')`
to zadzaiła to w ten sam sposób. Oba sposoby są dobre i niezadzaiłają gdy element nie zostanie znaleziony :)